swagger.json

```diff
@@ -4584,6 +4584,30 @@
                     "format": "int64",
                     "description": "The amount of driving time in violation in this cycle (in ms).",
                     "example": 50400000
+                  },
+                  "onDutyMsToday": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "The amount of on duty time today (in ms).",
+                    "example": 50400000
+                  },
+                  "driveMsToday": {
```

Comment:
Excellent point. I agree that there is a more complete representation of this data that we could provide. This change is to meet an explicit ask for these fields to be added so that there is parity with [fleet/drivers/{driver_id:[0-9]+}/hos_daily_logs](http://samsara-arcbest-api-preview-docs.s3-website-us-west-2.amazonaws.com/#operation/get_fleet_drivers_hos_daily_logs).